.github/workflows/master_branch_protection.yml

```diff
@@ -0,0 +1,74 @@
+name: PR to master branch from patch/release branch only
+
+on:
+  pull_request:
+    branches:
+    - master
```

Comment:
@Imipenem we gotta figure out a solution for this. Should def run on main!

Reply:
@Zethson according to GH actions docs this shouldn't be more complicated than simply adding `- main`.

I will check this out tomorrow!

Reply:
Still wish there was a "default" branch, but oh well